cupyx/jit/_typerules.py

```diff
@@ -132,4 +136,29 @@ def get_ctype_from_scalar(mode, x):
         if isinstance(x, complex):
             return _types.Scalar(numpy.complex64)
 
-    raise NotImplementedError(f'{x} is not supported as a constant.')
+    raise NotImplementedError(f'{x} is not scalar object.')
+
+
+_suffix_literals_dict = {
+    numpy.dtype('float64'): '',
+    numpy.dtype('float32'): 'f',
+    numpy.dtype('int64'): 'll',
```

Comment:
I am not sure, but could this cause problems in windows?
I think it should be ok, but double check is not bad. @leofang

Reply:
Prefer q over l whenever possible.

Reply:
Fixed in b84eb5f!